### ISSUE TITLE
Create "action" button on map screen

### ISSUE THREAD

**Issue by canyouswim** [canyouswim/Caching-on-Kai#53]
this will be a breaking change for users, and will necessitate a move in version numbering to v2.0 as a result.

This feature is to fundamentally change how the user interacts with the map screen.  previously clicking the OK/Enter button would center the map on the user's GPS position.  This change will:
-  Alter the numeric 5 key to toggle between centering on the selected cache (if any) and centering on the user's GPS position
- When the user clicks the OK/Enter button now, they will be presented with a menu of options.  Initial option choices will be:
1. Create a waypoint at the selected location
2. if the selected point is X distance away from a loaded geocache, then the user will be prompted to open that geocache details page

other options may be added to this list later

**Comment by canyouswim** [comment]
@david-p-s - i'm working on some updates to the create waypoint function, which i'm now linking to the OK/Enter button (the waypoint options are still there under the options menu).  A few things i'm working on in the "11/17-chris" branch:
- tweak the creation of the waypoint container in the cache list so we create a new/separate waypoint container for each waypoint created

you might want to wait on doing more work on the waypoints until i get through this bit of work

**Comment by canyouswim** [comment]
this is done - i'll create a separate issue for reviewing the new work!
